app/viewmodels/DeclarationDetailViewModel.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import helpers.Formatters
+import models.{CashAccount, CustomsDuty, Declaration, ExciseDuty, ImportVat}
+import models.domain.EORI
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.Aliases.{HtmlContent, SummaryList, SummaryListRow, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.{Key, Value}
+import utils.Utils.emptyString
+
+case class DeclarationDetailViewModel(eori: EORI, account: CashAccount)
+
+object DeclarationDetailViewModel {
+
+  def declarationSummaryList(declaration: Declaration)(implicit messages: Messages): SummaryList = {
+    SummaryList(
+      attributes = Map("id" -> "mrn"),
+      rows = Seq(
+        SummaryListRow(
```

Comment:
Could it be one for another time, or not at all, but would it be much work to modify the SummaryListRow to assume actions is none by default? It would let you remove them from all these constructors and the again in the next file.

I think all you'll need to is update SummaryList Row as follows:
actions: Option[Actions] = None